packages/core/src/initializer/npm-client.ts

```diff
@@ -65,7 +65,7 @@ export class NpmClient {
       const response = await this.innerNpmClient.get<NpmSearchResult>(path);
       return handleResult(path)(response);
     } catch (err) {
-      this.log.error(`Unable to reach 'https://registry.npmjs.com' (for query ${path}). Please check your internet connection.`, errorToString(err));
+      this.log.error(`Unable to reach '${getRegistry()}' (for query ${path}). Please check your internet connection.`, errorToString(err));
```

Comment:
Instead of using `getRegistry` here, you can simply let the npm registry location be injected, see previous comment about dependency injection